packages/quantic/force-app/main/default/lwc/quanticModal/quanticModal.js

```diff
@@ -0,0 +1,56 @@
+import {LightningElement, api} from 'lwc';
+
+/**
+ * The `QuanticModal` component provides the logic to easily implement a modal in Quantic. This component handles the animation logic, exposes methods to open and close the modal, offers the option to open the modal in full screen or just to cover the search interface, and exposes a set of slots to fully customize the modal content.
+ *
+ * @category Search
+ * @example
+ * <c-quantic-modal full-screen animation="slideToLeft"></c-quantic-modal>
```

Comment:
I think this example might be a good place to show usage of the available slots.

Reply:
Sure thing👌 I'll add that.